frontend/src/components/common/header/components/search-bar/search-bar.tsx

```diff
@@ -13,34 +14,47 @@ type Props = {
 };
 
 const SearchBar: FC<Props> = ({ onSearch }) => {
-  const { control, errors } = useAppForm<SearchPayload>({
+  const { control, errors, getValues } = useAppForm<SearchPayload>({
     defaultValues: DEFAULT_SEARCH_PAYLOAD,
   });
 
-  const { field } = useFormControl({ name: 'search', control });
-  const { value } = field;
+  const { setValue: setOuterFormValue, getValues: getOuterFormValues } =
+    useForm({
```

Comment:
`useAppForm` is already imported `useForm` hook, maybe it would be better to enhance existing one?

Reply:
And use `watch` instead of `getValues`, calling `setValue` is redundant bcos it`s already handled by `control`